### ISSUE TITLE
Rainloop under active development? 

### ISSUE THREAD

**Issue by mandusm** [RainLoop/rainloop-webmail#2172]
Is this project still under active development? It's been a year since the last updates were made.

**Comment by daffydock** [comment]
It seems dead, or semi-dead. However, and if interested, there is an active fork called Snappymail. Don't get me wrong, I like Rainloop, I just worry about some issues that have as far as I can tell remain unfixed. 

https://github.com/the-djmaze/snappymail

**Comment by mandusm** [comment]
Well, you sir, is what we call a scholar and a gentlemen! 
Thanks for the redirect! Already switched over my deployment!

**Comment by Pofilo** [comment]
Whatever is your choice, be careful with Rainloop right now:

https://thehackernews.com/2022/04/unpatched-bug-in-rainloop-webmail-could.html?m=1

**Comment by daffydock** [comment]
Yeah, that company literally opened an issue here earlier and no one answered. I used Rainloop on my Nextcloud install yet since the issue is still open and the security flaw is still at play, I had to remove it from my install. Shame.  

As per your own article, SnappyMail does not suffer the same security issues.

**Comment by smac0** [comment]
Now there's a new version, but with the appendix "legacy" and I can't find a changelog anywhere nor any announcement explaining this. 
Can anyone make sense of this?

**Comment by Neustradamus** [comment]
I have done a comment here:
- https://github.com/RainLoop/rainloop-webmail/pull/2187#issuecomment-1233475036

--------------

Please use SnappyMail from @the-djmaze, we can thanks for this work!
- https://snappymail.eu/
- https://github.com/the-djmaze/snappymail/

Please note that SnappyMail supports SCRAM-SHA-* for connection, very good security:
- https://github.com/the-djmaze/snappymail/issues/182
- https://github.com/the-djmaze/snappymail/blob/master/snappymail/v/0.0.0/app/libraries/RainLoop/Model/Account.php

Linked to:
- https://github.com/RainLoop/rainloop-webmail/issues/2185
- https://github.com/RainLoop/rainloop-webmail/pull/2183
- https://github.com/RainLoop/rainloop-webmail/issues/2180
- https://github.com/RainLoop/rainloop-webmail/issues/2162
- https://github.com/RainLoop/rainloop-webmail/issues/2142
- https://github.com/RainLoop/rainloop-webmail/issues/2134
- https://github.com/RainLoop/rainloop-webmail/issues/1831
